addons/account/wizard/account_invoice_send.py

```diff
@@ -134,6 +125,15 @@ def _send_email(self):
                 if prioritary_attachments:
                     main_attachment = prioritary_attachments[0]
                     invoice.with_context(tracking_disable=True).sudo().write({'message_main_attachment_id': main_attachment.id})
+            # with_context : we don't want to reimport the file we just exported.
+            self.composer_id.with_context(no_new_invoice=True,
+                                          mail_notify_author=self.env.user.partner_id in self.composer_id.partner_ids,
+                                          mailing_document_based=True,
+                                          )._action_send_mail()
```

Comment:
I would perhaps take the opportunity of this commit to correctly indent this if you don't mind :)

```suggestion
            self.composer_id.with_context(
                no_new_invoice=True,
                mail_notify_author=self.env.user.partner_id in self.composer_id.partner_ids,
                mailing_document_based=True,
            )._action_send_mail()
```

Reply:
LGTM